src/styles/base.css

```diff
@@ -103,15 +125,12 @@ li > a {
   width: calc(100% - 8px);
   max-width: 720px;
   object-fit: cover;
-  padding: 8px;
+  padding: 20px;
```

Comment:
What's the motivation for this? It looks like a lot of padding to me now, but I don't feel too strongly

Reply:
Nice catch. Meant to use 10px to keep it consistent with the homepage